python/ql/src/analysis/TypeHierarchyFailure.qhelp

```diff
@@ -4,11 +4,12 @@
 <qhelp>
 <overview>
 <p>In order to analyse uses of a class, all its attributes need to be known. Without the full inheritance hierarchy this is impossible.
-This is an informational query only. 
+This query finds classes where type inference fails and gives some indication as to why that failure occurred.
 </p>
 
 <p>
 This is an informational query only, this query depends on points-to and type inference.
+Unlike normal queries it does not indicate any problem with the program being analyzed.
```

Comment:
Also consider adding a References section with links to: 

* https://lgtm.com/help/lgtm/python-extraction
* https://lgtm.com/help/lgtm/customizing-code-extraction
* https://help.semmle.com/wiki/display/SD/Customizing+Python+extraction

Reply:
This assumes that there is something wrong with extraction, which there probably isn't.

Reply:
Okay.